<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="Izaskun Mallona">
    <meta name="author" content="Izaskun Mallona">
    <meta name="keywords" content="Izaskun, Mallona, Gonzalez, R, GNU, linux, UZH, IGTP, TCGA, single-cell, RNA-seq, gene, expression">
    <link rel="icon" href="favicon.ico">

    <title>Izaskun Mallona</title>

    <!-- Custom styles for this template -->
    <link href="css/readable.css" rel="stylesheet">

  </head>

  <body>

    <!-- Fixed navbar -->
    <nav class="navbar navbar-inverse navbar-fixed-top">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="index.html">Izaskun Mallona</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav">
            <li><a href="index.html">Home</a></li>
            <!-- <li><a href="software.html">Repositories</a></li> -->
            <li class="active"><a href="publications.html">Publications</a></li>
            <li><a href="teaching.html">Teaching</a></li>
            <li><a href="contact.html">Contact</a></li>
          </ul>
        </div><!--/.nav-collapse -->
      </div>
    </nav>

    <div class="container theme-showcase" role="main">

      <br/>
      <br/>
      <div class="page-header">
	<h4>Profiles</h4>
	<ul>
	  <li><a href="https://scholar.google.com/citations?user=kMfxCIYAAAAJ" target="_blank">Google scholar</a></li>
	  <li><a href="http://orcid.org/0000-0002-2853-7526" target="_blank">ORCID</a></li>
          <li><a href="https://publons.com/researcher/600812/izaskun-mallona/" target="_blank">Publons</a></li>
	</ul>
        <h4>Non OA copies</h4>
	<ul>
	  <li><a href="https://bitbucket.org/imallona/imallona.bitbucket.org/src/master/data/" target="_blank">PDF repository</a></li>
	</ul>
        <h4>Publications</h4>
	<ol reversed=true>
          <li>Robinson, M., Cai, P., Emons, M., Gerber, R., Germain, P.L., Gunz, S., Luo, S., Moro, G., Sonder, E., Sonrel, A., Wang, J., Wissel, D., & Mallona, I.. (2024). Ten simple rules for collaborating with wet lab researchers for computational researchers. arXiv</li>
          
<li>Hruska-Plochan, M., Wiersma, V., Betz, K., Mallona, I., Ronchi, S., Maniecka, Z., Hock, E.M., Tantardini, E., Laferriere, F., Sahadevan, S., Hoop, V., Delvendahl, I., Pérez-Berlanga, M., Gatta, B., Panatta, M., Bourg, A., Bohaciakova, D., Sharma, P., De Vos, L., Frontzek, K., Aguzzi, A., Lashley, T., Robinson, M., Karayannis, T., Mueller, M., Hierlemann, A., & Polymenidou, M. (2024). A model of human neural networks reveals NPTX2 pathology in ALS and FTLD. Nature, 626(8001), 1073–1083.</li>


<li>Teske, M., Wertheimer, T., Butz, S., Zwicky, P., Mallona, I., Elling, U., Lancrin, C., Becher, B., Grosso, A., Baubec, T., & Schmolka, N. (2024). Targeted CRISPR-Cas9 screening identifies transcription factor network controlling murine haemato-endothelial fate commitment. bioRxiv.</li>


<li>Pérez‐Berlanga, M., Wiersma, V., Zbinden, A., De Vos, L., Wagner, U., Foglieni, C., Mallona, I., Betz, K., Cléry, A., Weber, J., Guo, Z., Rigort, R., Rossi, P., Manglunia, R., Tantardini, E., Sahadevan, S., Stach, O., Hruska‐Plochan, M., Allain, F., Paganetti, P., & Polymenidou, M. (2023). Loss of TDP‐43 oligomerization or RNA binding elicits distinct aggregation patterns. The EMBO Journal, 42(17).</li>


<li>Anthony Sonrel, Almut Luetge, Charlotte Soneson, Izaskun Mallona, Pierre-Luc Germain, Sergey Knyazev, Jeroen Gilis, Reto Gerber, Ruth Seurinck, Dominique Paul, Emanuel Sonder, Helena L. Crowell, Imran Fanaswala, Ahmad Al-Ajami, Elyas Heidari, Stephan Schmeing, Stefan Milosavljevic, Yvan Saeys, Serghei Mangul, & Mark D. Robinson (2023). Meta-analysis of (single-cell method) benchmarks reveals the need for extensibility and interoperability. Genome Biology, 24(1).</li>


<li>Bottes, S., Jaeger, B., Pilz, G.A., Jörg, D., Cole, J., Kruse, M., Harris, L., Korobeynyk, V., Mallona, I., Helmchen, F., Guillemot, F., Simons, B., & Jessberger, S. (2020). Long-term self-renewing stem cells in the adult mouse hippocampus identified by intravital imaging. Nature Neuroscience, 24(2), 225–233.</li>


<li>Izaskun Mallona, Ioana Mariuca Ilie, Ino Dominiek Karemaker, Stefan Butz, Massimiliano Manzo, Amedeo Caflisch, & Tuncay Baubec (2020). Flanking sequence preference modulates de novo DNA methylation in the mouse genome. Nucleic Acids Research.</li>


<li>Berta Martín, Stella Pappa, Anna D\iez-Villanueva, Izaskun Mallona, Joaqu\'\in Custodio, Mar\'\ia José Barrero, Miguel A. Peinado, & Mireia Jordà (2020). Tissue and cancer-specific expression of DIEXF is epigenetically mediated by an Alu repeat. Epigenetics, 15(6-7), 765–779.</li>


<li>Manca, P., Mallona, I., Santini, D., Tonini, G., Rolfo, C., Robinson, M., & Pantano, F. (2019). A new bioinformatic pipeline allows the design of small, targeted gene panels for efficient TMB estimation. Annals of Oncology, 30, ii8.</li>


<li>Tao Y, Kang B, Petkovich DA, Bhandari YR, In J, Stein-O'Brien G, Kong X, Xie W, Zachos N, Maegawa S, Vaidya H, Brown S, Chiu Yen RW, Shao X, & Easwaran H (2019). Aging-like Spontaneous Epigenetic Silencing Facilitates Wnt Activation, Stemness, and BrafV600E-Induced Tumorigenesis.. Cancer cell.</li>


<li>Izaskun Mallona, Susanna Ausso, Anna Diez-Villanueva, Victor Moreno, & Miguel A. Peinado (2018). DNA co-methylation networks outline the structure and remodeling dynamics of colorectal cancer epigenome. bioRxiv.</li>


<li>Buj R, Mallona I, Díez-Villanueva A, Zafón C, Mate JL, Roca M, Puig-Domingo M, Reverter JL, Mauricio D, Peinado MA, & Jordà M (2018). Kallikreins stepwise scoring reveals three subtypes of papillary thyroid cancer with prognostic applications.. Thyroid : official journal of the American Thyroid Association.</li>


<li>Mallona I, Sierco A, & Peinado MA (2018). The Pancancer DNA Methylation Trackhub: A Window to The Cancer Genome Atlas Epigenomics Data.. Methods in molecular biology (Clifton, N.J.).</li>


<li>Mallona, I., & Peinado, M. (2017). Truke, a web tool to check for and handle excel misidentified gene symbols. BMC Genomics, 18(1).</li>


<li>Mallona, I., Díez-Villanueva, A., Martín, B., & Peinado, M. (2017). Chainy: an universal tool for standardized relative quantification in real-time PCR. Bioinformatics, 33(9), 1411–1413.</li>


<li>Jordà, M., Díez-Villanueva, A., Mallona, I., Martín, B., Lois, S., Barrera, V., Esteller, M., Vavouri, T., & Peinado, M. (2016). The epigenetic landscape of Alu repeats delineates the structural and functional genomic architecture of colon cancer cells. Genome Research, 27(1), 118–132.</li>


<li>Buj, R., Mallona, I., Díez-Villanueva, A., Barrera, V., Mauricio, D., Puig-Domingo, M., Reverter, J., Matias-Guiu, X., Azuara, D., Ramírez, J., Alonso, S., Rosell, R., Capellà, G., Perucho, M., Robledo, M., Peinado, M., & Jordà, M. (2016). Quantification of Unmethylated Alu (QUAlu): a tool to assess global hypomethylation in routine clinical samples. Oncotarget, 7(9), 10536–10546.</li>


<li>Mallona, I., Jordà, M., & Peinado, M. (2016). A knowledgebase of the human Alu repetitive elements. Journal of Biomedical Informatics, 60, 77–83.</li>


<li>Carrió, E., Díez-Villanueva, A., Lois, S., Mallona, I., Cases, I., Forn, M., Peinado, M., & Suelves, M. (2015). Deconstruction of DNA methylation patterns during myogenesis reveals specific epigenetic events in the establishment of the skeletal muscle lineage. STEM CELLS, n/a–n/a.</li>


<li>Terry, M., & Mallona, I. (2015). First record of albinism in salema, Sarpa salpa (Perciformes: Sparidae), in the Mediterranean Sea. Marine Biodiversity Records, 8.</li>


<li>Díez-Villanueva, A., Mallona, I., & Peinado, M. (2015). Wanderer, an interactive viewer to explore DNA methylation and gene expression data in human cancer. Epigenetics & Chromatin, 8(1).</li>


<li>Mallona, I., Díez-Villanueva, A., & Peinado, M. (2014). Methylation plotter: a web tool for dynamic visualization of DNA methylation data. Source Code for Biology and Medicine, 9(1).</li>


<li>Weiss, J., Mallona, I., Gomez-di-Marco, P., Fernández-Valera, J., & Egea-Cortines, M. (2012). Genotyping< i> Antirrhinum</i> commercial varieties using miniature inverted-repeat transposable elements (MITEs). Scientia Horticulturae, 144, 161-167.</li>


<li>Mallona, I., Egea-Cortines, M., & Weiss, J. (2011). Conserved and Divergent Rhythms of Crassulacean Acid Metabolism-Related and Core Clock Gene Expression in the Cactus Opuntia ficus-indica   . Plant Physiology, 156(4), 1978–1989.</li>


<li>Mallona, I., Weiss, J., & Egea-Cortines, M. (2011). pcrEfficiency: a Web tool for PCR amplification efficiency prediction. BMC Bioinformatics, 12(1).</li>


<li>Mallona, I., Lischewski, S., Weiss, J., Hause, B., & Egea-Cortines, M. (2010). Validation of reference genes for quantitative real-time PCR during leaf and flower development in Petunia hybrida. BMC Plant Biology, 10(1).</li>

          
        </ol>
      </div>
      
      <footer>
	<div class="row">
          <div class="col-lg-12">
            <p>Theme by <a href="http://thomaspark.co" rel="nofollow">Thomas Park</a>. Based on <a href="http://getbootstrap.com" rel="nofollow">Bootstrap</a>. Icons from <a href="http://fortawesome.github.io/Font-Awesome/" rel="nofollow">Font Awesome</a>. Web fonts from <a href="http://www.google.com/webfonts" rel="nofollow">Google</a>.</p>
          </div>
	</div>
      </footer>

    </div> <!-- /container -->


    <!-- Bootstrap core JavaScript
	 ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script>window.jQuery || document.write('<script src="assets/js/vendor/jquery.min.js"><\/script>')</script>
      <script src="dist/js/bootstrap.min.js"></script>
      <script src="assets/js/docs.min.js"></script>
      <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
      <script src="assets/js/ie10-viewport-bug-workaround.js"></script>
  </body>
</html>
